NARFO_BE/NARFO_BE/Controllers/ActivitiesController.cs

```diff
@@ -0,0 +1,119 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using NARFO_BE.Models;
+
+namespace NARFO_BE.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class ActivitiesController : ControllerBase
+    {
+        private readonly narfoContext _context;
+
+        public ActivitiesController(narfoContext context)
+        {
+            _context = context;
+        }
+
+        // GET: api/Activities
+        [HttpGet]
+        public async Task<ActionResult<IEnumerable<Activity>>> GetActivity()
+        {
+            return await _context.Activity.ToListAsync();
+        }
+
+        // GET: api/Activities/5
+        [HttpGet("{id}")]
+        public async Task<ActionResult<Activity>> GetActivity(int id)
+        {
+            var activity = await _context.Activity.FindAsync(id);
+
+            if (activity == null)
+            {
+                return NotFound();
+            }
+
+            return activity;
+        }
+
+        // PUT: api/Activities/5
+        [HttpPut("{id}")]
+        public async Task<IActionResult> PutActivity(int id, Activity activity)
+        {
+            if (id != activity.ActiviryId)
+            {
+                return BadRequest();
+            }
+
+            _context.Entry(activity).State = EntityState.Modified;
+
+            try
+            {
+                await _context.SaveChangesAsync();
+            }
+            catch (DbUpdateConcurrencyException)
+            {
+                if (!ActivityExists(id))
+                {
+                    return NotFound();
+                }
+                else
+                {
+                    throw;
```

Comment:
Why are you throwing? Are there any better ways of doing this?

Reply:
this auto generated code.... we'll change everything accordingly when we work on those particular controllers if that's ok

Reply:
Well I'm not sure what to say on this...

- You can leave it like this.
- You can remove the auto-generated code (sounds like its not being used)
- Or you can come up with another way instead of just throwing.

Choice is yours.

Reply:
We are using the code just for testing the relationships for tables in the database for now,  when that is done then we'll work on specific controllers, that's when all your standards will come in handy. We taking notes.. 

- We'll have try catch for everything could potentially throw error 500, and handle that in a manner that won't crush the app
- Indentation fixed, no white-spaces 
- Follow the same standards on if statements
- For helper classes, we'll prolly divide our project into DataAccess layer, API and business logic layer.... What do you think....


Are we good?